includes/REST/WithdrawController.php

```diff
@@ -579,16 +579,20 @@ public function batch_items( $request ) {
      * @return data
      */
     public function prepare_item_for_response( $withdraw, $request ) {
-        $methods = dokan_withdraw_get_methods();
+        $methods     = dokan_withdraw_get_methods();
+        $vendor      = $this->get_user_data( $withdraw->get_user_id() );
```

Comment:
$this->get_user_data is unsafe if actual vendor data is not found (probably user has been deleted), kindly try to rewrite it and change accordingly